### PR TITLE
13279 update choices to use gettext_lazy

### PR DESCRIPTION
**Pull request by arthanson**

### CODE REVIEW

**Comment by jeremystretch** [comment]
While I appreciate the consistency, I'm not sure it makes sense to provide translation support for things like interface types and wireless channels. We might want to stick to the arbitrary strings like statuses and colors.

File: netbox/dcim/choices.py [hunk 1]
```diff
@@ -120,8 +121,8 @@ class RackElevationDetailRenderChoices(ChoiceSet):
     RENDER_SVG = 'svg'
 
     CHOICES = (
```
Comment (jsenecal) [inline netbox/dcim/choices.py:123]: Do we need to translate these ?

File: netbox/utilities/choices.py [hunk 2]
```diff
@@ -222,8 +223,8 @@ class ImportFormatChoices(ChoiceSet):
     YAML = 'yaml'
 
     CHOICES = [
-        (AUTO, 'Auto-detect'),
-        (CSV, 'CSV'),
-        (JSON, 'JSON'),
-        (YAML, 'YAML'),
+        (AUTO, _('Auto-detect')),
+        (CSV, _('CSV')),
```
Comment (jsenecal) [inline netbox/utilities/choices.py:227]: Again, do we want to support translating file formats ?

**Review by jsenecal — COMMENTED** [review]
I added a few notes about supporting translation on things that perhaps shouldn't be but otherwise this looks good to me. Thanks @arthanson

File: netbox/utilities/choices.py (hunk 2 above)
Comment (jeremystretch) [reply netbox/utilities/choices.py:227]: I've made the presumption elsewhere that acronyms like JSON, YAML, CSV, and so on do _not_ need to be translated. In this specific case, we'd want to translate the "auto-detect" option however.

Comment (jsenecal) [reply netbox/utilities/choices.py:227]: I agree, my comment started on L227+

**Review by jeremystretch — APPROVED** [review]

File: netbox/dcim/choices.py (hunk 1 above)
Comment (jeremystretch) [reply netbox/dcim/choices.py]: No; these are just file extensions.

**Pull request merged**